backend/moonstreamapi/providers/bugout.py

```diff
@@ -278,7 +278,7 @@ def previous_event(
         return self.entry_event(search_results.results[0])
 
 
-class EthereumTXPoolProvider(BugoutEventProvider):
+class TXPoolProvider(BugoutEventProvider):
```

Comment:
Why did you change this name? We will be introducing a `PolygonTXPool` provider in the future.